### DISCUSSION TITLE
Loop out still showing Initiated status after 2 weeks

### DISCUSSION THREAD

**Discussion by jc-jnet** [Ride-The-Lightning/RTL#836] [Troubleshooting Questions] [answered]
A loop out request carried out using RTL on 3 October 2021 is still showing status Initiated.

Here is a [screenshot](https://drive.google.com/file/d/1gxOjEaDotPIAyXkVC1QbynTHVlH9VOwq/view?usp=sharing)

RTL is running as an app on umbrel.

A loop out request worked successfully on 2 October 2021. For some reason, this one on 3 October 2021 has got stuck at initiated status.

**Comment by saubyk** [comment, marked as answer]
Hi @jc-jnet do you see any Active HTLCs on the channels page?
if not, it's likely that the Loop out request has timed out, but for some reason not reflecting the status correctly. You can try restarting your umbrel once, so that loop server restarts and updates the status correctly.

If you can login to your umbrel via command line, you can try running `loop monitor` command which might provide you status of your loop out command.

**Comment by jc-jnet** [comment]
Thanks. I did not see any active HTLCs. I noticed that the channel with
Wallet of Satoshi node has become inactive. My other channels are active.

Not sure why thus one became inactive. I have been talking to them on Slack
about the loop out issue.

I have now rebooted umbrel.
